game/scripts/npc/abilities/bristleback_warpath.txt

```diff
@@ -42,7 +42,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "max_stacks"                                      "5 7 9 11 13"
+        "max_stacks"                                      "5 7 9 9 9"
```

Comment:
You said you can hardly reach 13 stacks. Then why leaving `11 13` max stacks is an issue?

Reply:
get an aghs and you are perma 13 stacks. the fact bb's don't go aghs worries me actually

Reply:
Because its pointlessly hard to mantain 11+ stacks. Why lina talent got changed from +1 fiery soul stack to +30/2% Bonus Per stack a while ago

Reply:
Again why is leaving this as it is an issue? Isn't this a nerf for Bristlebacks that buy Aghs?

Reply:
The Bonus damage you get from Warpath is pretty good. I just made getting the old damage values much easier. Also an Ultimate shouldn't have unachieveable stacks without aghs. (Unless the aghs upgrades warpath.) You get exactly the same damage "675 1170" but it is much more achieveable and easier to matain.

Reply:
Also whats the point in leaving pointless numbers on the spell just because it was their before.

Reply:
If anything I would rather increase the damage per stack at OAA levels or increase the 25 talent then just give him "11 13 stacks"

Reply:
I would put `10 11` instead but 9 is fine I guess.

Reply:
I increased the damage to be more then its old values. This passive will be much better now.